warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -178,27 +179,21 @@ def go_forward(self, browser_instance=None):
 
     def check_url(self, url):
         """To check whether the user provided url is valid or not."""
-        status = True
         search_http = re.search("http", url)
         if not search_http:
-            print_error("Provide the url along with http/https")
-            status = False
-            return status, url
+            print_warning("The url with http/https is missing")
+            return None
         try:
             url_open = urllib2.urlopen(url)
             get_status_code = url_open.code
             pattern = re.compile('^2[0-9][0-9]$')
             if not pattern.match(str(get_status_code)):
-                status = False
+                print_warning("The Status code for url : {} is {}".format(url, get_status_code))
         except urllib2.HTTPError as http_error:
             print_error("URLError: {} reason: ({}) status code: {}".format(url, http_error.reason, http_error.code))
-            status = False
         except urllib2.URLError as url_err:
             print_error("URLError: {} reason: ({})".format(url, url_err.reason))
-            status = False
-        if status == False:
-            print_error("Incorrect URL provided")
-        return status, url
```

Comment:
Hi @brettleehari,
Changing the number of arguments being returned is not advisable as it breaks backward compatibility for anyone making use of this function outside of the WSelenium keywords.